gnomad_constraint/resources/resource_utils.py

```diff
@@ -24,13 +25,30 @@
 DATA_TYPES = ["context", "exomes", "genomes"]
 MODEL_TYPES = ["plateau", "coverage"]
 GENOMIC_REGIONS = ["autosome_par", "chrx_nonpar", "chry_nonpar"]
+
+CUSTOM_VEP_ANNOTATIONS = ["transcript_consequences", "worst_csq_by_gene"]
```

Comment:
I replaced `cumstom_model` with `custom_vep_annotations` and added docstrings for this variable. My understanding is that the difference between them is they uses different vep annotations. Is this name accurate enough?

Reply:
This looks good to me